### ISSUE TITLE
map.on('load',..)  does not work on safari 10.0.1

### ISSUE THREAD

**Issue by fxi** [mapbox/mapbox-gl-js#3472]
**mapbox-gl-js version**: v0.26.0
### Steps to Trigger Behavior
- map.on("load",function(){alert("load")})
- examples :
  - https://www.mapbox.com/mapbox-gl-js/example/geojson-markers/
  - https://jsfiddle.net/r0o5hf8r/2/
### Expected Behavior

apply a function when the map is fully loaded
### Actual Behavior

map remains blank until a map move event occurs. No message is shown in console.

![safari_bug_10 0 1_demo](https://cloud.githubusercontent.com/assets/1196833/19835724/0b2bdf56-9e8f-11e6-95a7-0fff1abde44b.gif)

**Comment by mourner** [comment]
Interesting, does this only happen on Safari 10.0.1? I can't reproduce on 10.0.

**Comment by fxi** [comment]
I've tested on safari 10.0.1 with saucelab on macOS 10.11 (El-capitan) and it worked as expected. I'm still using Yosemite (macOS 10.10).

It seems to be very specific to my computer, but it's the first time I've got inconsistencies across browsers with mapbox-gl-js.

![safari_bug_10 0 1_demo](https://cloud.githubusercontent.com/assets/1196833/19835724/0b2bdf56-9e8f-11e6-95a7-0fff1abde44b.gif)

I guess we can close this issue if nobody can reproduce it. 

Thanks :)
